### ISSUE TITLE
Only Cryptonight benchmarks working on a RX 570 

### ISSUE THREAD

**Issue by gomesthiago** [nicehash/NiceHashMiner#752]
I'm using NHM Legacy 1.8.2 v2 and returned to the  17.11.1 driver, since the GPU isnt listed in newer drivers.

I am trying to benchmark the GPU but all scrypts/miners fails besides XmrStak and Claymore_old, both for cryptonight. All the rest ar
[log.txt](https://github.com/nicehash/NiceHashMinerLegacy/files/1580999/log.txt)
e sgminer ( X11Gost, Pascal, Libry, Decred,DoggerHashimoto ), and none of them work.

I have already reinstalled NHML, tried downloading bin files manually, tried letting the NHML download them automatic, but I doenst work either way. Also there isn't any equihash miners.

**Comment by Roomerkind** [comment]
Curious to see what happens with this as I am waiting on a power adaptor so I can hook up my brand new 570.

**Comment by S74nk0** [comment]
Issue related to old nhm version/build. Please use latest version and open a new issue for bugs and/or features.
